renderer/components/preferences/item/switch.js

```diff
@@ -48,6 +48,27 @@ class Switch extends React.Component {
             border: 1px solid #ccc;
             background-color: #fff;
           }
+
+          .loading:after {
+            border: none;
+            background: #fff url('data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTAwJSIgaGVpZ2h0PSIxMDAlIiB2aWV3Qm94PSIwIDAgMTYgMTYiCiAgICAgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KICAgICAgICA8c3R5bGU+CiAgICAgICAgICAKICAgIEBrZXlmcmFtZXMgc3Bpbm5lciB7CiAgICAgICAgMCUgewogICAgICAgICAgICBzdHJva2UtZGFzaG9mZnNldDogMTAuNTY7CiAgICAgICAgfQogICAgICAgIDUwJSB7CiAgICAgICAgICAgIHN0cm9rZS1kYXNob2Zmc2V0OiA1MC4yNDsKICAgICAgICB9CiAgICAgICAgMTAwJSB7CiAgICAgICAgICAgIHN0cm9rZS1kYXNob2Zmc2V0OiAwLjY2OwogICAgICAgIH0KICAgIH0KCgoKICAgICAgICAgIAogICAgICAgICAgICAKCiAgICAgICAgICBjaXJjbGUgewogICAgICAgICAgICAgIGZpbGw6IHRyYW5zcGFyZW50OwogICAgICAgICAgICAgIHN0cm9rZTogIzAwN2FmZjsKICAgICAgICAgICAgICBzdHJva2UtbGluZWNhcDogcm91bmQ7CiAgICAgICAgICAgICAgc3Ryb2tlLWRhc2hhcnJheTogY2FsYygzLjE0cHggKiAxNik7CiAgICAgICAgICAgICAgc3Ryb2tlLWRhc2hvZmZzZXQ6IDE2OwogICAgICAgICAgICAgIGFuaW1hdGlvbjogc3Bpbm5lciAzcyBsaW5lYXIgaW5maW5pdGU7CiAgICAgICAgICB9CiAgICAgIDwvc3R5bGU+CiAgICAgICAgPGNpcmNsZSBjeD0iOCIgY3k9IjgiIHI9IjciIHN0cm9rZS13aWR0aD0iMSIgZmlsbD0ibm9uZSIgPjwvY2lyY2xlPgogICAgPC9zdmc+Cg==') center;
```

Comment:
See https://css-tricks.com/probably-dont-base64-svg/ You don't have to Base64 encode it. Just put the SVG source right in there. All you have to do is to run it through `encodeURIComponent`.